### ISSUE TITLE
Possible sponsors

### ISSUE THREAD

**Issue by ghost** [rust-gamedev/wg#128]
Rusty Jam(Game jam with games written in rust) co-host here. We recently got approached by a crypto company wanting this:

> We are currently looking for interesting partnerships with in order to bring more amazing builders into the blockchain space. Would you be interested to catch up on aa short call to see where we could collaborate?
(and they did clarify that it's about Rusty Jam)

I don't exactly know what they want from us, but I think it would be great if they're going to be ready to sponsor Rusty Jam #3 prizes. I have zero experience with prizes, but I think the prize should be something small but will still make them feel good thing, like maybe 100$ dollars. Bigger prize is going to turn the whole thing from light-hearted competiotion to a battlefiled.

What do you think?

**Comment by Lokathor** [comment]
I don't think that this working group, or any part of the rust project, should have anything to do with Blockchain or Cryptocurrency anything. It's destructive to the planet, and to society. Since rustaceans live on the planet, and are members of society, it's harmful to all rustaceans.

**Comment by facorread** [comment]
> I don't think that this working group, or any part of the rust project, should have anything to do with Blockchain or Cryptocurrency anything. It's destructive to the planet, and to society. Since rustaceans live on the planet, and are members of society, it's harmful to all rustaceans.

I second that. Additionally, crypto lends itself to scams that are far too frequent and lead to significant legal consequences.

https://twitter.com/web3isgreat

**Comment by 64kramsystem** [comment]
Crypto associations will be controversial, independent of one's position on the topic.

A small reward like 100$ is a very good idea, and given the small amount (it's 1 or 2 hours of the salary of a senior SWE), probably (my guess), companies that already support Rust Gamedev, will be able to sponsor, without any associated controversy.

**Comment by 17cupsofcoffee** [comment]
I feel like the Rust community is largely (and rightly) negative on crypto/blockchain - for example, see the reaction to the [recent Rust Foundation tweet](https://twitter.com/rust_foundation/status/1537752005267136514) promoting a blockchain startup. Taking a sponsorship from a company like that is likely to put off as many people as the prize would attract, if not more.

**Comment by Voodlaz** [comment]
Hmm. Well I guess if we're going to do that decision nobody would really like it.
